CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+## 0.21.4
+
+### Miscellaneous
+
+- Fixed integ tests [#328](https://github.com/pulumi/pulumi-pulumiservice/issues/328)
```

Comment:
I think the result was that this was not just fixing the tests? There were meaningful bug fixes included in this change. Can we please capture what we fixed for end users?

Reply:
Updated to include bug https://github.com/pulumi/pulumi-pulumiservice/issues/319, which was also fixed during the PR that fixed integ tests.